CONTRIBUTING.md

```diff
@@ -6,6 +6,27 @@ All contributors (with GitHub accounts) will be recognized by being added to the
 
 If you're new to open source, you'll probably find [this open source guide](https://opensource.guide/how-to-contribute) useful.
 
+---
+
+## Table of Contents
```

Comment:
All markdown files automatically have a table of contents if you read them on GitHub https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/. Do you think it's still worth having this? Might be annoying to maintain

Reply:
That is fair, I never noticed that functionality. If that is the case I can drop the table of contents commit.